site/src/Librecores/ProjectRepoBundle/Entity/OrganizationMember.php

```diff
@@ -0,0 +1,204 @@
+<?php
+namespace Librecores\ProjectRepoBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * An Organization Member
+ *
+ * An organization member is a mapping between a User and an Organization he belongs.
+ *
+ * @ORM\Table("OrganizationMember")
+ * @ORM\Entity
+ * @ORM\HasLifecycleCallbacks
+ */
+class OrganizationMember
+{
+    const PERMISSIONS_REQUEST = 'REQUEST';
+    const PERMISSIONS_DENY    = 'DENY';
+    const PERMISSIONS_SUPPORT = 'SUPPORT';
+    const PERMISSIONS_MEMBER  = 'MEMBER';
+    const PERMISSIONS_ADMIN   = 'ADMIN';
```

Comment:
nit: these constants just name a single permission, e.g. PERMISSION_REQUEST. Feel free to ignore if you disagree.